core/partitioning/simpleDataPacker.go

```diff
@@ -0,0 +1,71 @@
+package partitioning
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/marshal"
+)
+
+// SimpleDataPacker can split a large slice of byte slices in chunks <= maxPacketSize
```

Comment:
you have the same commentary as sizeDataPacker - what is the difference - explain it in the comments. exaplin that it is not doing continuous marshalling. Add comment in the sizeDataPacker why is it doing continuous marshalling - precision, etc.

Reply:
added explanations